### ISSUE TITLE
'torch_geometric.transforms' has no attribute 'ToDevice'

### ISSUE THREAD

**Issue by akul-goyal** [rusty1s/pytorch_geometric#3074]
## Environment

* OS: Mac 11
* Python version: 3.7
* PyTorch version: 1.9
* CUDA/cuDNN version:
* GCC version:
* How did you try to install PyTorch Geometric and its extensions (wheel, source): Pip
* Any other relevant information:

## Checklist

- [X ] I followed the [installation guide](https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html).
- [ X] I cannot find my error message in the [FAQ](https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html#id1).
- [ ] I set up CUDA correctly and can compile CUDA code via `nvcc`.
- [ ] I do have multiple CUDA versions on my machine.

## Additional context
The pytorch geometric transform folder on my computer does not have the to_device.py that the git repo does. In fact, many of the files that are present on the gitrepo's transform folder are not present within the install library on my computer. I am trying to run the ARGVA example.

**Comment by rusty1s** [comment]
Yes, some files changed in current master. You need to install from master or use an [older version of the example](https://github.com/rusty1s/pytorch_geometric/blob/1.7.2/examples/argva_node_clustering.py).
